### PR TITLE
Merge Stable to master post 4.7.0 release

### PR DESCRIPTION
**Pull request by McdonaldSeanp**

### CODE REVIEW

**Comment by branan** [comment]
The travis failure is the FACT-1497 issue (and I still don't know how that got merged the first time). I'm in favor of ignoring that, since there are no other issues.

File: lib/puppet/version.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 
 module Puppet
-  PUPPETVERSION = '4.7.0'
+  PUPPETVERSION = '4.7.1'
```
Comment (Magisus) [inline lib/puppet/version.rb]: Is this the correct version number? Do we bump this separately after this mergeup?

Comment (branan) [inline lib/puppet/version.rb]: Yeah, we'll bump it as a separate PR

Comment (Magisus) [inline lib/puppet/version.rb]: 👍

**Comment by Magisus** [comment]
Looks good to me 👍

**Pull request merged**